tools/appveyor.psm1

```diff
@@ -451,24 +451,26 @@ function Invoke-AppveyorFinish
 
         if ($env:APPVEYOR_REPO_TAG_NAME)
         {
-            # ignore the first part of semver, use the preview part
-            $preReleaseVersion = ($env:APPVEYOR_REPO_TAG_NAME).Split('-')[1]
+            $preReleaseVersion = "v$($env:APPVEYOR_REPO_TAG_NAME)"
```

Comment:
Isn't the tag name already has `'v'` prefixed?

Reply:
I don't know - I can not test. I am only tying to follow previous logic.

Reply:
@TravisEz13 can you comment on this one? What would `$env:APPVEYOR_REPO_TAG_NAME` be like?
@iSazonov I think the next commit you submit should be marked with '[Feature]', so that we can see how the `nupkg` generation works in full build.

Reply:
[Feature] - done.

Reply:
> contains tag name for builds started by tag; otherwise this variable is undefined
I don't think we ever do this, unless AppVeyor counts a push with a tag.

Reply:
I see. So if an AppVeyor build is triggered by pushing a tag, then this environment variable will be set to be the tag.
So we don't need to prefix with a `'v'` here,